cmd/sonobuoy/app/gen.go

```diff
@@ -229,99 +223,3 @@ func getClient(kubeconfig *Kubeconfig) (*kubernetes.Clientset, error) {
 
 	return client, kubeError
 }
-
-// getConfig generates a config which has the the following rules:
-//  - command line options override config values
-//  - plugins specified manually via flags specifically override plugins implied by mode flag
-//  - config values override default values
-// NOTE: Since it mutates plugin values, it should be called before using them.
-func (g *genFlags) resolveConfig() *config.Config {
-	if g == nil {
-		return config.New()
-	}
-
-	conf := config.New()
-	suppliedConfig := g.sonobuoyConfig.Get()
-	if suppliedConfig != nil {
-		mergeConfigs(suppliedConfig, conf)
-		conf = suppliedConfig
-	}
-
-	// Resolve plugins.
-	//  - If using the plugin flags, no actions needed.
-	//  - Otherwise use the supplied config and mode to figure out the plugins to run.
-	//    This only works for e2e/systemd-logs which are internal plugins so then "Set" them
-	//    as if they were provided on the cmdline.
-	// Gate the logic with a nil check because tests may not specify flags and intend the legacy logic.
-	if g.genflags == nil || !g.genflags.Changed("plugin") {
-		// Use legacy logic; conf.SelectedPlugins or mode if not set
-		if conf.PluginSelections == nil {
-			modeConfig := g.mode.Get()
-			if modeConfig != nil {
-				conf.PluginSelections = modeConfig.Selectors
```

Comment:
This actually should have been pulled out and put in https://github.com/vmware-tanzu/sonobuoy/pull/1343 but that finishes the job and removes the selectors (this is all that it was used for).

Just putting an FYI here